controllers/gatekeeper_controller.go

```diff
@@ -60,14 +66,18 @@ var (
 		"~g_v1_serviceaccount_gatekeeper-admin.yaml",
 		"rbac.authorization.k8s.io_v1_clusterrole_gatekeeper-manager-role.yaml",
 		"rbac.authorization.k8s.io_v1_clusterrolebinding_gatekeeper-manager-rolebinding.yaml",
-		"rbac.authorization.k8s.io_v1_role_gatekeeper-manager-role.yaml",
 		"rbac.authorization.k8s.io_v1_rolebinding_gatekeeper-manager-rolebinding.yaml",
 		auditFile,
 		webhookFile,
 		"~g_v1_service_gatekeeper-webhook-service.yaml",
 		validatingWebhookConfiguration,
 	}
-	validationGatekeeperWebhook = "validation.gatekeeper.sh"
+	openshiftAssets = []string{
+		"rbac.authorization.k8s.io_v1_role_gatekeeper-manager-role.yaml",
+	}
+	vanillaAssets = []string{
+		"rbac.authorization.k8s.io_v1_role_gatekeeper-manager-role.yaml",
```

Comment:
I think we should keep this in its original location instead of within a new directory because it's generated. The openshift specific one can remain in its own directory since it's not generated and we'll just have to maintain it.